docs/design_proposals/design-proposal-template.md

```diff
@@ -13,7 +17,8 @@ Please provide an rationale covering following points:
 
 1. Why is this required?
 2. If its re-design, What are cons with current implementation?
-3. Are there any another work-around and if yes, why not keep using it.
+3. Is there any another work-around and if yes, why not keep using it.
+4. Mention related issues, if there are any.****
```

Comment:
If the `****` is intentional, I don't get what it means

Reply:
no its unintentional.